mandatory/4-tax.js

```diff
@@ -17,7 +20,16 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(num) {
+  let final = ((num * 1.2).toFixed(2)).toString();
```

Comment:
```suggestion
  let final = ((num * 1.2).toFixed(2)).toString();
```

I think this task is hinting for you to reuse the function you just wrote. Look up at calculateSalesTax. How can you use that function again here? How would using the function make this code clearer?

I also think this variable name could be clearer. Why is naming variables clearly a great way to improve our code?